mlflow/gateway/providers/openai.py

```diff
@@ -74,22 +74,10 @@ def _add_model_to_payload_if_necessary(self, payload):
             return payload
 
     async def chat(self, payload: chat.RequestPayload) -> chat.ResponsePayload:
-        from fastapi import HTTPException
         from fastapi.encoders import jsonable_encoder
 
         payload = jsonable_encoder(payload, exclude_none=True)
         self.check_for_model_field(payload)
-        if "n" in payload:
-            raise HTTPException(
-                status_code=422, detail="Invalid parameter `n`. Use `candidate_count` instead."
-            )
-
-        payload = rename_payload_keys(
-            payload,
-            {"candidate_count": "n"},
-        )
-        # The range of OpenAI's temperature is 0-2, but ours is 0-1, so we double it.
-        payload["temperature"] = 2 * payload["temperature"]
```

Comment:
Are we no longer doing any casting / standardization of temperature parameters for any provider?

Reply:
we're using OpenAI's 0-2 now. Doh, need to update other 0-1 providers to scale down by 0.5

Reply:
Great catch!

Reply:
Fixed!